src/contracts.json

```diff
@@ -50,32 +50,29 @@
     },
     "version": "35a6cd3"
   },
-  "morden": {
-    "birthblock": 3757156,
+  "mordor": {
+    "birthblock": 0,
```

Comment:
Can we set it to the proper height of the tx(s) that created the contracts?

Reply:
this was set intentionally as Mordor just new chain and contract deployed is around 35000 block. Anyway. I have set the exact number